src/pages/[...locale]/collections/[slug].astro

```diff
@@ -85,14 +92,32 @@ const collectionHtml = await getMarkdownHtml(collection);
 						})
 					}
```

Comment:
The new `chapterList` behavior should ideally replace the above list of actual posts when it is present. In other words, `collectionPosts` should only be used when `chapterList` is not defined.